CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Terminal going transparent during visual bell
 - Selection not being cleared when sending chars through a binding
 - Mouse protocols/encodings not being mutually exclusive within themselves
+- Underline/Strikeout bleeds into current line on backspace
```

Comment:
Should I also update it to reflect all template fixes?

Reply:
I feel like we should phrase it more generally probably.